provider/gcp/gcp_instance.go

```diff
@@ -117,6 +118,61 @@ func (p *GCloud) CreateInstance(ctx *lepton.Context) error {
 		}
 	}
 
+	// Attach disk(s) upon instance creation if RunConfig.AttachVolumeOnInstanceCreate is set
+	for mntSrc, mntDst := range c.Mounts {
+		if !c.RunConfig.AttachVolumeOnInstanceCreate {
+			continue
+		}
+
+		// VirtFS VirtIO 9p mount - skip it
+		if strings.HasPrefix(mntSrc, "/") {
+			continue
```

Comment:
9P mounts are for onprem instances only, specifying a 9P mount directive for a GCP instance doesn't make sense, so I wouldn't do this check. If the user inserts a mntSrc with a leading '/' I guess it will result in a disk not found error